### PR TITLE
hamdi.html contribution

### PR DESCRIPTION
**Pull request by hamdihusein**
Made changes to hamdi.html

- Added fun facts
- Added image from online
- Pulling request to hopefully merge changes

### CODE REVIEW

File: your-turn/hamdi.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+</head>
+<body> 
+    <h1>Hamdi</h1>
+    <img src="https://legionofart.com/uploads/blackgirlmagic_508971.jpeg" class="side profile picture"> 
+
+    <div class="container">
+        <ul>
+            <li>My favorite color is purple</li>
+            <li>My favorite book is A Thousand Splendid Suns</li>
+            <li>My birthday is on a major holiday. Take a guess!</li>
```
Comment (tthompson899) [inline your-turn/hamdi.html:18]: Christmas? 👀

**Review by tthompson899 — APPROVED** [review]
Awesome work Hamdi! 
I wasn't sure if you wanted to use the image you added, it's not linked. If this is something you wanted to do, you can add Paris.jpg to the images folder and then change the image tag to: 
``

NOTE: 
- The `..` This will back out of the current directory you are in
- `/images/Paris.jpg` this will locate the image

Again, you don't have to change it if you don't want to - Totally up to you 😄 

Let me know if you're ready for me to merge this in.

File: your-turn/hamdi.html (hunk 1 above)
Comment (hamdihusein) [reply your-turn/hamdi.html:18]: Yup! 

Also, I made changes, saved them, and pushed it to the origin. I included the Paris image. It looks like it's showing up. Should I pull a new request or can you see those changes to be able to merge them? Thank you for your feedback!!

Comment (tthompson899) [reply your-turn/hamdi.html:18]: Nice!
Your changes look great! Since you already have an open pull request, you do not need to make a new one :)

**Pull request merged**